packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -183,8 +183,8 @@ const taskDefinition = new ecs.TaskDefinition(this, 'TaskDef', {
 Images supply the software that runs inside the container. Images can be
 obtained from either DockerHub or from ECR repositories, or built directly from a local Dockerfile.
 
-* `ecs.ContainerImage.fromDockerHub(imageName)`: use a publicly available image from
-  DockerHub.
+* `ecs.ContainerImage.fromInternet(imageName, { credentials?: MY_SECRET })`: use a public or private image from
```

Comment:
I wouldn't write it like this, this isn't copy/pasteable user code. Just add a new line below to do something like:

 `ecs.ContainerImage.fromInternet(imageName, { credentials: mySecret })`: use a private image that needs credentials.